envSetup.sh

```diff
@@ -312,14 +314,6 @@ InstallTunnelAppliance() {
     ./mstunnel-setup
 }
 
-SetupTunnelPrereqs() {
```

Comment:
Why is this getting removed? This needs to be done in order to install the certs before tunnel is installed.

Reply:
The copy commands now make those directories. The issue was on reset acme was getting wiped so the acme commands weren't running, so the copy commands had nothing to copy. 
As for the touch command I can add it in! It just wasn't being used before so I'm not sure if it was needed now.

Reply:
the touch command gets rid of the Eula check during install, so it is one less point of interaction. As for the certs, I don't see a change to the copy command, which should give an error saying that they couldn't create the regular file since the underlying directories don't exist.